### PR TITLE
ghost-writer-journey

### PR DESCRIPTION
**Pull request by Sparco7**
ghost writer journey links are completed

### CODE REVIEW

File: app/controllers/jobs_controller.rb [hunk 1]
```diff
@@ -12,16 +11,17 @@ def new
   end
 
   def create
+    @request = find_request
```
Comment (arosty) [inline app/controllers/jobs_controller.rb]: here and same for line 9: change the line from '@request = find_request' to 'find_request'
Then change line 43 to: @request = Request.find(params[:request_id])

File: app/controllers/requests_controller.rb [hunk 2]
```diff
@@ -1,8 +1,35 @@
...
+              'Engineering']
+
 class RequestsController < ApplicationController
   before_action :find_request, only: [:show, :update, :destroy]
 
   def index
-    @requests = Request.all
+    @subj = SUBJ
+    # raise
+    if params[:search]
```
Comment (arosty) [inline app/controllers/requests_controller.rb]: on-liner: @requests = params[:search] ? Request.where(subject: params[:search]) : Request.all

File: app/views/requests/index.html.erb [hunk 3]
```diff
@@ -4,26 +4,12 @@
...
-        <li>Computer science</li>
-        <li>Sociology</li>
-        <li>Statistics</li>
-        <li>Linguistics</li>
-        <li>Philosophy</li>
-        <li>Modern languages</li>
-        <li>Theology</li>
-        <li>Engineering</li>
+        <% @subj.each do |subject|  %>
+          <li><%= link_to subject, requests_path(:search => subject) %></li>
```
Comment (arosty) [inline app/views/requests/index.html.erb]: change it to modern way: requests_path(search: subject)

File: app/views/requests/index.html.erb [hunk 4]
```diff
@@ -4,26 +4,12 @@
...
+        <% end%>
```
Comment (arosty) [inline app/views/requests/index.html.erb]: space missing (sorry haha)

File: app/views/requests/show.html.erb [hunk 5]
```diff
@@ -18,6 +18,7 @@
       <% if policy(@request).edit? %>
         <p><%= link_to 'update', request_path(@request), method: :patch %></p>
       <% end %>
+      <p><%= link_to 'apply', new_request_job_path(@request) %></p>
```
Comment (arosty) [inline app/views/requests/show.html.erb]: capitalize --> 'Apply'

**Comment by arosty** [comment]
Very nice! Sorry for my petty comments

File: app/controllers/jobs_controller.rb (hunk 1 above)
Comment (Sparco7) [reply app/controllers/jobs_controller.rb]: done

File: app/controllers/requests_controller.rb (hunk 2 above)
Comment (Sparco7) [reply app/controllers/requests_controller.rb]: its more readble like that

File: app/views/requests/index.html.erb (hunk 3 above)
Comment (Sparco7) [reply app/views/requests/index.html.erb]: done

File: app/views/requests/index.html.erb (hunk 4 above)
Comment (Sparco7) [reply app/views/requests/index.html.erb]: done

File: app/views/requests/show.html.erb (hunk 5 above)
Comment (Sparco7) [reply app/views/requests/show.html.erb]: done

**Pull request merged**